tests/ui/test_monitoring.py

```
@@ -95,3 +109,103 @@ def test_monitoring(app, agent_idref_data, script_info):
+    assert "DB missing" in res.output
+
+
+def test_monitoring_print_missing_error(app, script_info):
+    """print_missing with unknown doc_type outputs the error message."""
+    runner = CliRunner()
+    res = runner.invoke(es_db_missing_cli, ["xxx", "-d", "0"], obj=script_info)
+    assert "xxx" in res.output
+    assert "Error" in res.output
+
+
+def test_monitoring_info_difference_db_es(app, agent_idref_data, script_info):
```

Ruff (0.15.1)

[warning] Unused function argument: `app`

(ARG001)